content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,39 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. The
+workspace is the directory containing _user visible_ part of your
+<abbr>project</abbr> e.g. raw datasets, source code, ML models, etc. Users work
```

Comment:
- _user visible_ -> just _visible_ ?
- "e.g. raw data..." -> "raw data, ... models, etc. that you're currently using." perhaps

Reply:
To my taste _visible_ is cleaner. But someone can say _visible to whom?_ to this and we may need to explain it. 

_Raw data_ is better, yes. 

Thanks.

Reply:
"Visible part of the project" seems perfectly OK to me.

> Raw data is better

You already had "raw data" 🙂 My suggestion is to add "that you're currently using."

Reply:
I thought you meant dataset ➡️  data 😆 

"that you're currently using" seems a bit redundant to me but let's see what happens.

Reply:
One of the points of DVC projects is to track multiple versions of data,models,etc. so pointing out that the workspace is where you find the current versions seems pretty key to me. But open to suggestions on other ways to emphasize that.

Reply:
How about the changes in 44af0a9f ?

Reply:
Hey sorry for the delay... Reading now.

Reply:
> can have multiple versions of data, models, and other kinds of artifacts within the workspace

This doesn't sound right. The workspace only has one version of the data, etc. at a time.

Reply:
And the paragraph became really long again!